client/src/components/Page/ChooseYourRunPage.js

```diff
@@ -23,37 +24,33 @@ const ScrollContainer = styled.div`
   }
 `;
 export default function ChooseYourRunPage() {
+  const [runs, setRuns] = useState(null);
+  useEffect(() => {
+    async function fetchData() {
+      const newRuns = await getRuns();
+      setRuns(newRuns);
+    }
+    fetchData();
+  }, []);
+
   return (
     <>
       <ChooseYourRunPageContainer>
         <AppHeader title={"Choose Your Run"} />
         <ScrollContainer>
-          <Link to="/created_run">
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-              isFavorite={true}
-              runName={"Severin Brücke Run"}
-            ></RunCard>
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-              runName={"Grüngürtel"}
-            ></RunCard>
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-            ></RunCard>
-
-            <RunCard
-              date={"44.44.444"}
-              time={"14:44"}
-              distance={"10"}
-            ></RunCard>
-          </Link>
+          {runs?.map((run) => (
+            <Link key={run.id} to={`/runs/${run.id}`}>
+              <RunCard
+                date={run.date}
+                time={run.time}
+                distance={run.distance}
+                isFavorite={true}
+                runName={run.runName}
```

Comment:
```suggestion
                isFavorite={true}
                {...run}
```

you could simplify your code a bit 😄